core/src/main/resources/hudson/PluginManager/tabBar.jelly

```diff
@@ -31,7 +31,7 @@ THE SOFTWARE.
     <l:tab name="${%Updates}" active="${page=='updates'}" href="." />
     <l:tab name="${%Available}" active="${page=='available'}" href="./available" />
     <l:tab name="${%Installed}" active="${page=='installed'}" href="./installed" />
-  <j:if test="${h.hasPermission(app.pluginManager.UPLOAD_PLUGINS) || h.hasPermission(app.pluginManager.CONFIGURE_UPDATECENTER)}">
+  <j:if test="${h.hasPermission(app.ADMINISTER) }">
```

Comment:
This check shouldn't exist anymore, as the entire plugin manager is protected by Overall/Administer.